docs/gaia/gaiacli.md

```diff
@@ -130,29 +130,24 @@ multi signature account see [Multisig Transactions](#multisig-transactions).
 
 ### Fees & Gas
 
-Each transaction may either supply fees or gas prices, but not both. Most users
-will typically provide fees as this is the cost you will end up incurring for
-the transaction being included in the ledger.
+Each transaction may either supply fees or gas prices, but not both. 
 
 Validator's have a minimum gas price (multi-denom) configuration and they use
-this value when when determining if they should include the transaction in a block
-during `CheckTx`, where `gasPrices >= minGasPrices`. Note, your transaction must
-supply fees that are greater than or equal to __any__ of the denominations the validator requires.
+this value when when determining if they should include the transaction in a block during `CheckTx`, where `gasPrices >= minGasPrices`. Note, your transaction must supply fees that are greater than or equal to __any__ of the denominations the validator requires.
 
 __Note__: With such a mechanism in place, validators may start to prioritize
-txs by `gasPrice` in the mempool, so providing higher fees or gas prices may yield
-higher tx priority.
+txs by `gasPrice` in the mempool, so providing higher fees or gas prices may yield higher tx priority.
 
 e.g.
 
 ```bash
-gaiacli tx send ... --fees=100photino
+gaiacli tx send ... --fees=1000000uatom
```

Comment:
This suggests a fee of 1atom which seems very high. Shall we use a `uatom` value that actually represents ~$0.05?

Reply:
y

Reply:
still needs to be updated I think